sdl2promela/stop_condition/translator.py

```diff
@@ -1112,6 +1183,29 @@ def _generate_filter_out_alternative(
         expression = _generate(context, statements[index].expression)
         expression = _add_precheck_expressions(context, expression)
         expressions.append(expression)
+    # For all observers with ignore state generate list of expressions
+    for observer in context.observers_with_ignore:
+        ignorestates = context.processes[observer].ignorestates
+        expression = _create_observer_state_check(ignorestates, observer)
+        expressions.append(expression)
+
+    if context.observers_with_success:
```

Comment:
I don't understand why it is here. Maybe the host function should receive a comment?

Reply:
the comment below was extended